scripts/common.py

```diff
@@ -12,7 +12,7 @@
 import multiprocessing
 
 from pathlib import Path
-from typing import List
+from typing import Any, Dict, List, NoReturn, Optional, Tuple, Union
```

Comment:
What Python version do we use?  Newer ones can use `|` for `Union`.

Reply:
Ancient on old distros in CI, so we can't rely on anything remotely new.